src/api/app/mailers/event_mailer.rb

```diff
@@ -58,6 +58,7 @@ def set_event_headers
     return unless @event
 
     headers['X-OBS-event-type'] = @event.template_name
+    headers['List-ID'] = "OBS Notification System <#{@event.template_name}.#{URI.parse(@configuration.obs_url).host.downcase}>"
```

Comment:
Instead of `List-ID`, you can use `X-Loop` instead. That's [what the Red Hat Bugzilla does](https://bugzilla.redhat.com/show_bug.cgi?id=1689640).

Reply:
What's the advantage over List-ID? for List-ID we can make more specific dynamic lists, that would be the same for X-Loop (depending on the exact RFC definition of it) or more coarse (like simply the instance hostname).

Reply:
You will never implement a functioning filter system in a mail header. If someone wants a functioning mail filter, get off GMail...

Reply:
so X-BeenThere: $projectname? I'm not sure that will work (I haven't tried).